app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -137,6 +138,21 @@ data class OverdueAppointment(
         scheduledAfter: LocalDate
     ): PagingSource<Int, OverdueAppointment>
 
+    @Query("""
+      SELECT * FROM OverdueAppointment
+      WHERE 
+        IFNULL(patientAssignedFacilityUuid, appt_facilityUuid) = :facilityUuid 
+        AND (appt_scheduledDate < :scheduledBefore AND appt_scheduledDate > :scheduledAfter)
+        AND (appt_remindOn < :scheduledBefore OR appt_remindOn IS NULL)
+        GROUP BY appt_patientUuid
+        ORDER BY isAtHighRisk DESC, appt_scheduledDate DESC, appt_updatedAt ASC
+    """)
+    fun overdueInFacilityPagingSource2(
```

Comment:
Can we rename the old DAO method that we gonna remove to `overdueInFacilityPagingSource_old` and this to `overdueInFacilityPagingSource`, please? 

We have started avoiding using numbers to indicate 2 different variants.